app/graphql/mutations/integration_collection_mappings/netsuite/create.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Mutations
+  module IntegrationCollectionMappings
+    module Netsuite
+      class Create < BaseMutation
+        include AuthenticableApiUser
+        include RequiredOrganization
+
+        graphql_name 'CreateNetsuiteIntegrationCollectionMapping'
+        description 'Create Netsuite integration collection mapping'
+
+        input_object_class Types::IntegrationCollectionMappings::Netsuite::CreateInput
+
+        type Types::IntegrationCollectionMappings::Netsuite::Object
```

Comment:
I am not sure why do we have `netsuite` specific object to return. IMO we should use generic object which can be used for other integrations as well